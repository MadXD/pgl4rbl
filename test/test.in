client_address=89.97.188.34
helo_name=trinity.develer.com

client_address=8.8.8.8
helo_name=dns1.google.com

client_address=8.8.8.8
helo_name=google

client_address=89.97.188.34
helo_name=[ciao]

client_address=89.97.188.34
helo_name=[255.256.1024.12]

