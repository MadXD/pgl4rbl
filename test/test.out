action=permit You are cleared to land

action=permit You are cleared to land

action=defer Are you a spammer? If not, just retry!

action=defer Are you a spammer? If not, just retry!

action=defer Are you a spammer? If not, just retry!

